design/user_types.go

```diff
@@ -32,10 +32,24 @@ var UpdateWorkItemPayload = Type("UpdateWorkItemPayload", func() {
 	Required("type", "fields", "version")
 })
 
+// CreateWorkItemTypePayload explains how input payload should look like
 var CreateWorkItemTypePayload = Type("CreateWorkItemTypePayload", func() {
-	Attribute("name", String, "Readable name of the type like Task, Issue, Bug, Epic etc.")
-	Attribute("fields", HashOf(String, fieldDefinition), "Type fields those must be followed by respective Work Items.")
-	Attribute("extendedTypeName", String, "If newly created type extends any existing type")
+	Attribute("name", String, "Readable name of the type like Task, Issue, Bug, Epic etc.", func() {
+		Example("Epic")
+	})
+	Attribute("fields", HashOf(String, fieldDefinition), "Type fields those must be followed by respective Work Items.", func() {
+		Example(map[string]interface{}{
+			"system.administrator": map[string]interface{}{
+				"Type": map[string]interface{}{
+					"Kind": "string",
+				},
+				"Required": true,
+			},
+		})
+	})
+	Attribute("extendedTypeName", String, "If newly created type extends any existing type", func() {
+		Example("(optional field)Parent type name")
```

Comment:
Yeah it is not that real, but if
1. I do not put anything, goagen will put some random value.
2. If I put `NoExample()` then it goagen will put `"-"` (will this be fine?)
3. And if I put empty string it is again not real value.
And I want user to know that it is optional and can be skipped while creating the item.